network/p2pNetwork.go

```diff
@@ -258,7 +258,29 @@ func (n *P2PNetwork) RequestConnectOutgoing(replace bool, quit <-chan struct{})
 }
 
 // GetPeers returns a list of Peers we could potentially send a direct message to.
-func (n *P2PNetwork) GetPeers(options ...PeerOption) []Peer { return nil }
+func (n *P2PNetwork) GetPeers(options ...PeerOption) []Peer {
```

Comment:
Spoke with @cce offline but logging here anyway. This looks like it should work but empirically only adding peers for PeersConnectedOut and PeersConnectedIn doesn't fix the test but adding it for all peer options does. 

Tracking it down now but we should hold off on merging until we figure it out.